src/modules/repository/implementation/sequelize/migrations/20241215122200-create-paranet-assets.js

```diff
@@ -0,0 +1,60 @@
+export async function up({ context: { queryInterface, Sequelize } }) {
+    await queryInterface.createTable('paranet_assets', {
+        id: {
+            type: Sequelize.INTEGER,
+            primaryKey: true,
+            autoIncrement: true,
+        },
+        blockchain_id: {
+            type: Sequelize.STRING,
+            allowNull: false,
+        },
+        ual: {
+            type: Sequelize.STRING,
+            allowNull: false,
+        },
+        paranet_ual: {
+            type: Sequelize.STRING,
+            allowNull: false,
+        },
+        public_assertion_id: {
+            type: Sequelize.STRING,
+            allowNull: true,
+        },
+        private_assertion_id: {
+            type: Sequelize.STRING,
+            allowNull: true,
+        },
+        sender: {
+            type: Sequelize.STRING,
+            allowNull: true,
+        },
+        transaction_hash: {
+            type: Sequelize.STRING,
+            allowNull: true,
+        },
+        error_message: {
```

Comment:
This was used when there was multiple records for failed, do you think we should need it